build/innosetup/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ubuntu:18.04
+
+RUN dpkg --add-architecture i386
+RUN apt-get update && apt-get install -y --no-install-recommends curl ca-certificates wine-stable wine32 xvfb
+
+ENV WINEDEBUG fixme-all
+ENV WINEARCH win32
+
+# # Install Inno Setup binaries
+RUN curl -SL "http://www.xs4all.nl/~mlaan2/ispack/innosetup-5.5.9.exe" -o is.exe
```

Comment:
Please use official URL with link following and NOT direct mirror!
For example: `curl -SL "https://github.com/jrsoftware/issrc/releases/download/is-5_6_1/innosetup-5.6.1.exe" -o is.exe` for GitHub Release